frontend/src/searchform/searchform.tsx

```diff
@@ -7,26 +7,13 @@ import { useTipsySelector } from '../tipsyselector/tipsyselector';
 
 const INGREDIENT_STORE = "get-me-drunk-fridge";
 
-const DEFAULT_INGREDIENTS = [
-    "carbonated water",
-    "coffee",
-    "cola",
-    "crushed ice",
-    "cow's milk",
-    "drinking water",
-    "ice",
-    "sugar",
-    "milk",
-    "water",
-];
-
 function SearchForm({
     submit
 }: {
     submit(query: { weight: number, ingredients: IIngredient[], promille: number }): void
 }) {
     const [weight, setWeight] = React.useState(70);
-    const [ingredients, setIngredients] = React.useState<IIngredient[]>([]);
+    const [ingredients, setIngredients] = React.useState<IIngredient[] | null>(null);
     const [tipsySelectorEl, promille] = useTipsySelector({ rangeOptions: { min: .3, max: 2 } });
 
     // If the ingredients were loaded before the user started adding some, 
```

Comment:
As you've removed the default ingredients, this comment no longer holds true.

Reply:
true, we will touch these parts anyways soon though, so I just leave it that way